docs/APDB.md

```diff
@@ -0,0 +1,47 @@
+Alert Production Database (APDB) Schema
+=======================================
+
+The [apdb](./python/lsst/sdm_schemas/schemas/apdb.yaml) schema describes the Alert Production Database (APDB).
+This database is updated by the [Alert Production Pipeline](https://github.com/lsst/ap_pipe) and used for nightly [ap_verify](https://github.com/lsst/ap_verify) runs within continuous integration.
+Previous processing runs may differ from the current schema.
```

Comment:
I don't think you need to explain what APDB is and how it is populated in this document.